recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
-{% set name = "nest-simulator" %}
-{% set version = "2.18.0" %}
+{% set name = "nesttest" %}
```

Comment:
This is not good @steffengraber :(

It will create new packages like https://anaconda.org/conda-forge/nesttest immediately without a trace from where it's coming from. Please do not use branches for testing.

Reply:
@steffengraber are you ok, if I remove this repository from anaconda.org?

Reply:
Ok, please delete!

Reply:
Done. Please never use a branch on this feedstock, if you are not planning to maintain a different version.